model/installation_request.go

```diff
@@ -433,6 +459,22 @@ func (p *PatchInstallationDeletionRequest) Apply(installation *Installation) boo
 	return applied
 }
 
+func (p *PatchInstallationRequest) addMissingIngressSourceRanges(installation *Installation) string {
+	ips := strings.Split(*p.AllowedRanges, ",")
```

Comment:
This could panic if AllowedRanges is `nil` for whatever reason - I know you're checking in the calling function, but I think it's worth checking here in case someone else uses the function elsewhere

Reply:
@nickmisasi nice point